packages/api-client/src/conversation/ConversationAPI.ts

```diff
@@ -480,6 +480,15 @@ export class ConversationAPI {
     return response.data;
   }
 
+  public async postForClients(clientId: string, conversationId: string): Promise<ClientMismatch> {
+    try {
+      await this.postOTRMessage(clientId, conversationId);
+      throw Error(`Expected backend to throw error.`);
```

Comment:
```suggestion
```

Reply:
It should. The call should never pass.

Reply:
Alright! But let's add the `new` keyword then :)
```suggestion
      throw new Error(`Expected backend to throw error.`);
```